docs/conf.py

```diff
@@ -53,10 +53,20 @@
     "source_edit_link": "https://github.com/canonical/imagecraft",
 }
 
+# Sitemap configuration: https://sphinx-sitemap.readthedocs.io/
+html_baseurl = "https://canonical-imagecraft.readthedocs-hosted.com/"
+
+if "READTHEDOCS_VERSION" in os.environ:
+    version = os.environ["READTHEDOCS_VERSION"]
+    sitemap_url_scheme = "{version}{link}"
```

Comment:
I don't know whether the current guidance for adding a meta-sitemap for versions is fully hashed-out, but let's see if this gets us where we want.